content/en/learn/scaling/mindmap.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Scaling InnerSource"
+subtitle: "How can I scale InnerSource?"
+type: "community"
+image: "images/learn/patterns.png"
+weight: 1
+---
+
+{{< markmap path="markmap_content.md" >}}
```

Comment:
@jeffabailey all the other files on the repo use a `-` as separator.
I suggest renaming `markmap_content.md` to `markmap-content.md` for consistency.